app/controllers/main/routes.py

```diff
@@ -65,7 +67,8 @@ def profilePage(username):
                                interests = interests,
                                interests_ids = interests_ids,
                                upcomingEvents = upcomingEvents,
-                               rsvpedEvents = rsvpedEvents)
+                               rsvpedEvents = rsvpedEvents,
+                               completedBackGroundCheck = completedBackGroundCheck.passBackgroundCheck)
```

Comment:
be consistent with `Background` vs `BackGround`

Reply:
changed to Background